tasks/task960_ancora-ca-ner_named_entity_recognition.json

```diff
@@ -0,0 +1,914 @@
+{
+    "Source": [
+        "BSC-TeMU/ancora-ca-ner"
+    ],
+    "Definition": "This task is about doing Named Entity Recognition(NER) for every word in a given Catalan sentence. First, split the sentence into single words or punctuation symbols. For each word, you need to assign an IOB tag. IOB tag consists of two parts separated by '-' or a single part depending on whether the word belongs to a bigger chunk or not. When breaking a bigger chunk, you can easily break a noun phrase into multiple words. For example, if we break 'New York', we get 'New' and 'York.' Here, both 'New' and 'York' belong to the bigger chunk 'New York,' which are chunks themselves. If the word doesn't belong to any bigger chunk, you will assign an 'O' tag to that particular word. But if it belongs to a bigger chunk, you will assign a two-part tag like this: The I- prefix before a tag indicates that the word is inside a bigger chunk. The B- prefix before a tag indicates that the word is the beginning of a bigger chunk that immediately follows another chunk within the same bigger chunk without O tag words between them. When a bigger chunk comes after an O tag, the first token of the chunk takes the I- prefix. So, suppose a token does belong to a bigger chunk. In that case, you need to assign 'LOC' or 'ORG' or 'PER' tag as the second part of the tag where 'LOC' indicates the token belongs to a location chunk, 'ORG' indicates the token belongs to an organization name chunk, 'PER' indicates the token belongs to a person name chunk, and 'MISC' indicates the token belongs to chunk which is not a person, location or an organization. Avoid using words from other languages.",
+  
+    "Positive Examples": [ 
+        { 
+            "input": "Una setantena de treballadors de Fomento de Construcciones y Contratas a Manresa , gairebé la totalitat de la plantilla , s' han manifestat aquest migdia pels carrers de la ciutat .",
+            "output": "O O O O O B-ORG I-ORG I-ORG I-ORG I-ORG O B-LOC O O O O O O O O O O O O O O O O O O O",  
```

Comment:
The output in this form looks a bit odd to a human annotator (they're hard to map to the original tokens). How about we change the outputs to sth like this? 

```
Una (O) setantena (O) de (O) treballadors (O) de (O) Fomento (B-ORG) de (I-ORG) Construcciones (I-ORG)....
```

Reply:
Or: 
```
Una  O,    setantena O,  de O,  treballadors O,  de O,  Fomento B-ORG,  ...
```

Reply:
Actually repeating the input in the output is redundant. We are assigning a tag to each token(words separated by space). I think it will be intuitive for the user. Swaroop suggested me to model the output this way too.That's my opinion. If you still want t change it let me know.